day_7/high_level.md

```diff
@@ -0,0 +1,26 @@
+# Cracking the Code: How to Build & Solve a Caesar Cipher with Ruby
```

Comment:
This is so thorough of an explanation it reads to me more like a lesson on how to code a caesar cipher than pseudocode. Very impressive.
As you approach new problems, recognize that pseduocoding (aka writing a step-by-step recipe not in code) can be a process for you to figure out unknowns for yourself.